bokeh/server/protocol_handler.py

```diff
@@ -99,7 +99,7 @@ def handle(self, message, connection):
         try:
             work = yield handler(message, connection)
         except Exception as e:
-            log.error("error handling message %r: %r", message, e)
+            log.error("error handling message %r: %r", message, e, exc_info=1)
             log.debug("  message header %r content %r", message.header, message.content, exc_info=1)
```

Comment:
I think the `exc_info=1` should now be removed from here, so it is not duplicated on the error and debug lines. Also, I think it would be good to stat in the error message that turning on debug output will show the entire message (then anyone who wants it has all the information they need right there)